rmgpy/rmg/model.py

```diff
@@ -1343,31 +1343,13 @@ def getStoichiometryMatrix(self):
                 nu = rxn.getStoichiometricCoefficient(spec)
                 if nu != 0: stoichiometry[i,j] = nu
         return stoichiometry.tocsr()
-
-    def addSeedMechanismToCore(self, seedMechanism, react=False):
+    
+    def constructLibraryReactions(self,library):
         """
-        Add all species and reactions from `seedMechanism`, a 
-        :class:`KineticsPrimaryDatabase` object, to the model core. If `react`
-        is ``True``, then reactions will also be generated between the seed
-        species. For large seed mechanisms this can be prohibitively expensive,
-        so it is not done by default.
+        adds reactions from a given kinetics library in preparation to add it to core as seed mechanism
+        or to the edge as a reaction library
```

Comment:
I don't think I was clear in my explanation earlier. We should discuss this in person before anything happens with this PR.